intro-to-R-tidyverse/03-intro_to_tidyverse.Rmd

```diff
@@ -8,15 +10,22 @@ editor_options:
   chunk_output_type: inline
 ---
 
-**CCDL 2020**
 
-## Objective for this notebook analysis: 
+## Objectives
+
+This notebook will demonstrate how to:
+
+- Use the tidyverse package to read and write data frames
```

Comment:
I was also torn because both ggplots and tidyverse notebooks address it but it appears to be in more detail here in the tidyverse notebook. We could include a very specific learning objective: "Read in data from text files" for the ggplot notebook, where the tidyverse notebook is more general and refers both to reading in data and saving?

Reply:
With the extra info in this notebook, I think it is fine to leave this off the ggplot notebook.